### PR TITLE
chore: add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by akurinnoy**
### What does this PR do?

Add the `CONTRIBUTING.md` file.

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/22243

### CODE REVIEW

**Review by ibuziuk — APPROVED** [review]
👍

**Review by olexii4 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+# Watch and build changes for the backend:
+yarn --cwd packages/dashboard-backend build:watch
+```
+
+```sh
+# Watch and build changes for the frontend:
+yarn --cwd packages/dashboard-frontend build:watch
+```
+
+Local backend is prerequisite for the development server. You can run the dev server using the following command:
```
Comment (dkwon17) [inline CONTRIBUTING.md:232]:
```suggestion
Local backend is a prerequisite for the development server. You can run the dev server using the following command:
```

**Review by dkwon17 — COMMENTED** [review]
I have some suggestions, looks great

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+You may opt to add a remote with `git add remote` and push to a personal fork rather than the upstream `eclipse-che/che-dashboard` repository.
+
+### Building an Image
+
+To build an image within a Che workspace on an OpenShift cluster, the cluster and OpenShift user must be configured to allow with rootless builds. Additionally, the following attribute must be added to the devfile:
+
+```yaml
+attributes:
+  controller.devfile.io/scc: container-build 
+```
```
[the remark this answers is not in the document]
Comment (akurinnoy) [reply CONTRIBUTING.md]: removed, thanks!

**Review by dkwon17 — APPROVED** [review]

**Pull request merged**